src/store.js

```diff
@@ -15,6 +15,21 @@ var store = {
     auth: false
 };
 
+
+store.resetStore = function() {
+    // TODO : I'm thinking we need some user/trader objects that manage this.
+    this.user = {
+        id: null,
+        traders : []
+    };
+    this.trader = {
+        id : null,
+        pendedVouchers : []
+    };
+    this.vouchers = this.vouchers.splice(0, this.vouchers);
```

Comment:
at this point it could contain a voucher that someone was going to submit, but somehow managed not to. :-/ the slice business just zeros it without upsetting vue's observers, which, to be fair i dont care much about at this point in the spa. I'm quite keen to convert to vuex next sprint!